src/operator/tensor/pseudo2DTranspose_op-inl.cuh

```diff
@@ -39,22 +39,29 @@ namespace mxnet {
 namespace op {
 namespace cuda {
 
-
-template <typename DType, typename CType>
+/*!
+ * \brief The `transpose_pseudo2D` based on chosen vectorized types. It transpose an array of
```

Comment:
@ptrendx I've also added the doc here for the `transpose_pseudo2D`. Correct me if you find any problem.